src/web/src/app/space/invitation-list/component.ts

```diff
@@ -219,11 +220,16 @@ export class SpaceInvitationListComponent implements OnInit {
 		}
 		this.space.approveInvitation(item.id)
 			.subscribe(_ => {
+				this.refreshParent();
 				if (this.paginator.pageIndex == 0) {
 					this.getItems(this.pageSize, 0);
 				} else {
 					this.paginator.firstPage();
 				}
 			});
 	}
+
+	refreshParent() {
+		this.refreshListComponentEvent.emit();
+	}
```

Comment:
В идеале компонент не должен знать о существовании каких то parent'ов. Если нужно, мы уведомляем о деяствиях которые мы совершаем, а тот кто нами управляет, на эти события реагирует.

```suggestion
		this.space.approveInvitation(item.id)
			.subscribe(_ => {
				this.goToFirstPage();
				this.onApprove.emit(item);
			});
	}

	goToFirstPage() {
		if (this.paginator.pageIndex == 0) {
			this.getItems(this.pageSize, 0);
		} else {
			this.paginator.firstPage();
		}
	}
```

Кажется в этом файле много мест где есть кусок кода из goToFirstPage, посмотри пожалуйста и замени по возможности на вызов goToFirstPage.